client/app/intake/components/NonratingRequestIssueModal.stories.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { MemoryRouter } from 'react-router';
+import { PAGE_PATHS } from '../constants';
+import ReduxBase from 'app/components/ReduxBase';
+import { reducer, generateInitialState } from 'app/intake';
+
+import NonratingRequestIssueModal from './NonratingRequestIssueModal';
+
+const RouterDecorator = (Story) => (
+  <MemoryRouter initialEntries={[PAGE_PATHS.ADD_ISSUES]}>
+    <Story />
+  </MemoryRouter>
+);
+
+const ReduxDecorator = (Story) => {
+  return <ReduxBase reducer={reducer} initialState={generateInitialState()}>
+    <Story />
+  </ReduxBase>;
+};
+
+const defaultArgs = {
+  intakeData: {
+    activeNonratingRequestIssues: []
+  },
+  featureToggles: {}
+};
```

Comment:
```suggestion
const defaultArgs = {
  intakeData: {
    activeNonratingRequestIssues: []
  },
  featureToggles: {},
  onSubmit: () => ''
};
```

If the modal is submitted an exception is currently thrown. It might be good to hand the component a dummy `onSubmit` method just to appease it.